### ISSUE TITLE
Black Face Syndrome is spreading

### ISSUE THREAD

**Issue by TheCampingOwl** [ForgottenGlory/Dungeons-Deviousness#77]
**D&D Version**
1.4.0

**Describe the bug**
The syndrome is spreading, Hulda, Alvor, Jon Battle-Born, Biki(riverwood companion) are now suffering of this syndrome, they were fine in 1.3.0

**To Reproduce**
Look at them

**Expected behavior**
Have a normal skin color

**Screenshots**
https://i.imgur.com/B8OlAlm.jpg

**Additional context**
No

**Comment by TheCampingOwl** [comment]
Gerdur in Riverwood as well.

**Comment by TheCampingOwl** [comment]
And Anoriath in Whiterun(market stalls NPC) as well as Nazeem.

**Comment by TheCampingOwl** [comment]
Whiterun Dragonsreach as well

Fianna
Gerda
Farengar Secret Fire

**Comment by celadoneiron** [comment]
I believe this is happening because the load order of plugins within D&D NPCs Merged is different to the install order in the left pane:

Install Order | Merge Load Order
-- | --
Fresh Faces - SSE | Cuyima 3DNPC – redone.esp
Kalilies NPCs | DawnguardNPCs.esp
Jampion NPCs - Male NPC Overhaul | Jampion NPCs – USSEP patch.esp
Male Npc Overhaul SE | Jampion NPCs.esp
Pandorable's NPCs | KaliliesNPCs.esp
Pandorable's NPCs - Dawnguard | LOS Combined.esp
Pandorable's NPCs - Dragonborn | male npc overahul se.esp
Pandorable's Black-Briar Ladies | MiraiRenewal.esp
Pandorable's Warrior Women | PAN_BlackBriar.esp
Pandorable's Lethal Ladies | PAN_LethalLadies.esp
Ladies of Whiterun NPC SSE | PAN_NPCs.esp
Ladies of Solitude NPC Replacer SSE | PAN_NPCs_DB.esp
Gman749 Ladies of Dawnguard NPC Replacer SSE | PAN_NPCS_DG.esp
Pandorable's Valerica | PAN_Serana.esp
Pandorable's Serana | PAN_Valerica.esp
Mirai Replacer | PAN_WarriorWomen.esp
Cuyima Interesting NPCs SE | Whiterun2.esp

They need to be the same (one way or another, either by rebuilding the merge or reordering the mods in the left pane, whichever gets the desired combination of overhauls).

Additionally, I think that any changes made by "Jampion NPCs - USSEP patch.esp" will be lost because "Jampion NPCs.esp" is being loaded after it within the merge.

**Comment by Keithht151** [comment]
Thought i would do a quick run through major cities and add to the list, sorry for possible copies, and i only checked in places that werent locked

Whiterun - Hulda/ Base=00013BA3/ Ref=0001A66E
          Jon BB/ Base=00013BBa/ Ref=0001A68A
          Anoriath/ Base=00013B97/ Ref=0001A680
          Gerda/ Base=000c247E/ Ref=000D16E1
          Fianna/ Base=000D16DD/ Ref=000D16E2
          Farengar/ Base=00013BBB/ Ref=0001A67E

Falkreath - Siddgeir/ Base=00013653/ Ref=00019822
            Delacourt/ Base00072663/ ref=00072667

Solitude - Corpulus Vinius/ Base=00013266/ Ref=000198A0

Dawnstar - Abelone/ Base=0008774F/ Ref=00087750

Winterhold - Korir/ Base=0001C188/ Ref=0001C1B5

Riften - Maul/ Base=000371D6/ Ref=000eBE26
         Harralk/ Base=00013361/ Ref=00019DEB

**Comment by celadoneiron** [comment]
The cause for most or maybe even all of these BFBs appears to be a combination of the merge-internal load order not matching the install order and the NPC visual data being overwritten by various patches. The most common culprit by far is know_your_armor_patch.esp - after fixing the load-install disparity (first by changing the install order to match the merge, then by reverting the install order to default and rebuilding the merge to match) and disabling the KYA patch, _every_ NPC listed by Keithht151 no longer had the BFB.

The KYA patch is also reverting changes made by the USSEP, among other things, so I think it need to be rebuilt and hand-checked. Other patches that override changes in D&D NPCs Merged.esp include the Bashed Patch and know_your_enemy_patch.esp, and there may well be others.

When I tried to rebuild the merge, I ran into an issue with the custom load order not actually saving. It would display as if it had worked, but then going in game with the problem patches disabled didn't resolve every BFB the way that swapping the install order around and disabling the patches did. Selecting and deselecting a plugin when editing the merge would reveal that all the plugins had reverted to the original load order, even when I tested by adding a new plugin that was totally unrelated to the rest. The only way I could make the load order inside the merge maintain was to adjust the priority of the component esps in the MO2 load order and then choose the zMerge option to use the game load order. After building the merge that way and making sure the KYA patch was disabled, the BFBs were gone.
